.github/workflows/scripts/add-labels.rb

```diff
@@ -0,0 +1,25 @@
+require "octokit"
+
+client = Octokit::Client.new(access_token: ENV["GITHUB_TOKEN"])
+
+repo = ENV["GITHUB_REPOSITORY"]
+pr_number = ENV["GITHUB_REF"].split("/")[2]
+
+pr_labels = {
+  no_bottles: 0
+}
+
+client.pull_request_files(repo, pr_number).each do |file|
```

Comment:
This function is called twice I see. Might be a good idea to call it once, always lesser GitHub API knocks.

Reply:
I think it'll work with just one call. Fixed.